src/helpers/constants.js

```diff
@@ -36,9 +36,39 @@ export const cevHelpTypes = {
     noNeedsIdentified: 'No needs identified'
 };
 
-export const TEST_AND_TRACE_FOLLOWUP_TEXT = 'test-and-trace-followup-text';
-export const TEST_AND_TRACE_FOLLOWUP_EMAIL = 'test-and-trace-followup-email';
+// some of these are included within the grouping object, but their usage is not changed to point to object within code.
+export const TEST_AND_TRACE_FOLLOWUP_TEXT = 'test-and-trace-followup-text'; // included into object
+export const TEST_AND_TRACE_FOLLOWUP_EMAIL = 'test-and-trace-followup-email'; // included into object
 export const PRE_CALL_MESSAGE_TEMPLATE = 'pre-call-message-template';
 export const SELF_ISOLATION_PRE_CALL_MESSAGE_TEMPLATE = 'self-isolation-pre-call-message-template';
-export const EUSS_PRE_CALL_MESSAGE_TEMPLATE = 'euss-pre-call-message-template';
+export const EUSS_PRE_CALL_MESSAGE_TEMPLATE = 'euss-pre-call-message-template'; // included into object
 export const LINK_WORK_BULK_MESSAGE_TEMPLATE = 'link-work-bulk-message-template';
+
+// currently contains duplicate, but ideally, after refactoring the other ones get
+// removed. The code becomes much clearer, when the constants are grouped.
+export const TEMPLATE_ID_ALIASES = {
```

Comment:
Do you mean this contains duplicate values? It doesn't appear to; or do you mean it's a repeat of a list of aliases that exist elsewhere in the code?

Reply:
Uhm... it's more of a note for any refactoring. I figured that it's probably best to group related environment variables into an enum-like structure _(like all the template alias names under one object; all the call types under another object)_ this simplifies down the imports & makes it easier to understand what type of data a variable belongs in.

The duplicates I was referring to are... the variables above: Ln 40, 41, 44. Upon creating an enum-like object (Ln49) I duplicated some of the variable names and values seen above _(So Ln 40 is duplicated in Ln54)_. I couldn't get rid of the ungrouped variables because that would mean changes accross the code base, where that variable is used. This would be better off done in a dedicated refactoring PR.